src/SDKs/DataFactory/Management.DataFactory/Generated/Models/SapCloudForCustomerLinkedService.cs

```diff
@@ -0,0 +1,118 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.DataFactory.Models
```

Comment:
@wenbof can you link the spec changes for the corresponding change that resulted in this new SDK generation.
Also if you use generate.cmd it will create the meta data file required to verify the generated SDK

Reply:
@shahabhijeet  the spec change is here: https://github.com/Azure/azure-rest-api-specs/pull/1946#pullrequestreview-73268739. is this what you need?

Reply:
I used autorest to generate the new SDK files and the results are all included in the change of this PR. are there anything else special I need to do? I checked the other like salesforce sink, it seems there are no extra steps either.

Reply:
@shahabhijeet , I have used the generate.cmd to create the meta data file. could you please double review? thanks.